src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -468,4 +468,18 @@ double irreducible_mass(double area);
  */
 double christodoulou_mass(double dimensionful_spin_magnitude,
                           double irreducible_mass);
+
+/*!
+ * \ingroup SurfacesGroup
+ * \brief Radial distance between two `Strahlkorper`s.
+ *
+ * \details Compute the pointwise radial distance between two Strahlkorpers with
+ * the same center, first (if necessary) prolonging or restricting
+ * `strahlkorper_b` to a Strahlkorper with the same angular resolution as
+ * `strahlkorper_a`.
```

Comment:
Instead of prolonging/restricting `stralhlkorper_b`, I think this instead should prolong whichever `strahlkorper` has the smallest resolution, i.e. the output would be the size of the larger Strahlkorper and not always the size of `strahlkorper_a`.    But I could be convinced otherwise.

Reply:
I decided to change the code to always prolong to the surface with more resolution

Reply:
Thank you!  Please change the docs above to reflect that change.